src/osx/cocoa/preferences.mm

```diff
@@ -149,6 +149,7 @@ void OnSelectPageForTool(const wxToolBarToolBase *tool)
 
         //   3. new page is shown and the title updated.
         info->win->Show();
+        info->win->Refresh();
```

Comment:
I think this really merits a comment as it's something that just begs to be removed (because in wx API `Show()` is not supposed to be followed by `Refresh()`).

Alternatively, perhaps it should be done in (generic-only?) `wxWindow::Show()` itself? Or is this really specific just to the preferences dialog?

Reply:
> Or is this really specific just to the preferences dialog?

That's the only situation where I noticed anything like this. Might make sense to put it in wxOSX's Show(), but most of the time it shouldn't be needed; this code does something weird in how it creates, hides, resizes, then re-shows the windows.